tests/upgrade/run.sh

```diff
@@ -54,17 +54,21 @@ test_upgrade() {
     wait_for_api
     setup_client_TLS_certs
 
+    # TODO(sbostick): try installing to secondary "secured cluster"
     info "Deploying sensor"
-    "$TEST_ROOT/$DEPLOY_DIR/sensor.sh"
-    validate_sensor_bundle_via_upgrader "$TEST_ROOT/$DEPLOY_DIR"
+    "$TEST_ROOT/$DEPLOY_DIR/sensor.sh"  # <---- override as "deploy/openshift"?
```

Comment:
This is going to be the rub. The API endpoint and other configuration for sensor<->central will be wrong. I would put some hold logic in here and work out of CI to get the power/z cluster configured as the 'remote' cluster in the GKE cluster that gets configured for test the CI run. 
My goto hold for openshift CI:
```
    touch /tmp/hold
    while [[ -e /tmp/hold ]]; do
        info "Holding this job for debug"
        sleep 60
    done

```

Reply:
when you are ready to run the test you can remove the hold through `oc rsh` https://docs.engineering.redhat.com/display/StackRox/Tips+for+using+OpenShift+CI